src/vector/jitsi/index.ts

```diff
@@ -27,6 +27,7 @@ import {
 import { ElementWidgetActions } from "matrix-react-sdk/src/stores/widgets/ElementWidgetActions";
 import { logger } from "matrix-js-sdk/src/logger";
 import { IConfigOptions } from "matrix-react-sdk/src/IConfigOptions";
+import { _t, setLanguage } from "matrix-react-sdk/src/languageHandler";
```

Comment:
we'll have to double check that this doesn't cause the bundle to rapidly expand, as it was a major concern last time we tried to add translations to the widget.

Reply:
Related: #12839

Reply:
Tree Shaking would be great, but that is not an easy change. Not sure if can refactor the language handle to be better useable. Building a custom translation solution for this widget doesn't feel right. I guess we have to postpone this PR.

Reply:
If we can better isolate the language handling code, I think that would be good. From inspecting the bundle it appears to be pulling in the js-sdk and a bunch of its dependencies, which would explain the size (the obvious indicator was the EventType enum).

Reply:
Looks like this will all be because of the `SettingsStore` call in the language functions

Reply:
It's not just `SettingsStore`, `ModuleRunner` also pulls everything in. If I remove both, the bundle size is back to the previous size.

The good thing is that we don't need to read/write the settings for the Jitsi widget. It would be good if the `SettingsStore` was just a wrapper interface around an actual implementation, either as a NoopSettingsStore for the Jitsi Widget, or the full store for the rest? The actual implementation could be inserted on startup. What do you think?

For `ModuleRunner`, I guess the problem comes from `ProxiedModuleApi`, but I haven't discovered the exact problem yet.